cli-commands/help.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 UbiCli.on("help") do
+  desc "Get program help"
```

Comment:
Very minor but `program` seems pretty generic here. Maybe `CLI` or `command` would be more direct/specific?

Reply:
I'll make that change.